src/components/_proxied-dot-io/consul/subnav/style.module.css

```diff
@@ -5,4 +5,17 @@
 
 .subnav {
 	border-top: 1px solid transparent;
+
+	& [data-ga-button='try-for-free'],
+	& [data-ga-button='try-hcp-consul'] {
+		color: transparent;
```

Comment:
Hide text on load and since we can't stack the buttons similar to the hero, set a min width equal to the greatest button width.